SampleBrowser/SampleBrowser/UI/Settings/SettingsViewController.swift

```diff
@@ -0,0 +1,84 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import WebKit
+
+/// The settings page
+class SettingsViewController: UIViewController, UITableViewDelegate {
+    private var tableView: UITableView
+    private var dataSource = SettingsDataSource()
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.font = UIFont.systemFont(ofSize: 30, weight: UIFont.Weight.medium)
+        label.text = "Settings"
+        label.textAlignment = .center
+    }
+
+    init() {
+        self.tableView = UITableView(frame: .zero, style: .grouped)
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTableView()
+        configureTitle()
+        tableView.reloadData()
+        view.backgroundColor = tableView.backgroundColor
+    }
+
+    private func configureTableView() {
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.isScrollEnabled = false
+        view.addSubview(tableView)
+
+        NSLayoutConstraint.activate([
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        tableView.tableHeaderView = UIView(frame: CGRect(origin: .zero,
+                                                         size: CGSize(width: 0, height: CGFloat.leastNormalMagnitude)))
```

Comment:
Nit: I'm still not entirely clear why we need to use `leastNormalMagnitude` instead of 0.0, I saw that you added a comment about this same question on the last PR; I think I was taking a closer look at that but then got sidetracked so I will go back and re-read it.

In any case it might be nice to add a brief comment here just to explain the reason since it could be unclear to our future selves what the purpose is. Not a big deal though.

Reply:
Mh, so it doesn't work to remove the `tableHeaderView` if we use `0`. I'm not sure why, but I will add a comment.